roles/tomcat9/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+# defaults file for roles/tomcat8
+tomcat8_enabled: true
+tomcat8_hostname: localhost
+tomcat8_server_port: 8005
+tomcat8_catalina_port: 8983
+tomcat8_catalina_redirect_port: 8443
```

Comment:
These vars are all named `tomcat8_` something - should we update them to be `tomcat9_`? I'm not clear if we're changing the existing role to install tomcat 9 instead of tomcat 8, or if we need to maintain both for a while, or what. But I find this confusing now, and I bet Future Me will find it even more confusing in future.

Reply:
we are replacing tomcat8. I was uncomfortable removing it because of the geoserver but will remove it with the next commit.